### ISSUE TITLE
Add a Time Interval Selector Row!

### ISSUE THREAD

**Issue by Sweeper777** [xmartlabs/Eureka#606]
A lot of the forms I see have this kind of selector, and the form that I'm creating needs this kind of row too. It's basically a picker view like this:

<img width="296" alt="screen shot 2016-08-12 at 7 48 45 am" src="https://cloud.githubusercontent.com/assets/13585047/17608762/5130645c-6061-11e6-89d7-945e8a6f91ab.png">

It lets the user select a period of time. This is very useful if the form has fields like "duration", "Remind me XXX later", where XXX is a period of time that the user selects. This can also be useful in a highly customisable timer app.

I know I can do it myself using custom rows but I find it hard to follow the guide in README. There's too many generic types and associated types and stuff. I am overwhelmed.

Can you add such a row?

**Comment by cmbasnett** [comment]
I also have need for this. I'm currently using the `CountDownPickerRow` as a hack to achieve something like this, however dealing with it's `value` is cumbersome.

**Comment by mats-claassen** [comment]
If you are having issues creating custom rows then you might find [this tutorial post](https://blog.xmartlabs.com/2016/09/06/Eureka-custom-row-tutorial/) useful. We will soon add a more advanced second part.

I have also created [this gist](https://gist.github.com/mats-claassen/fa8ec5834eeb34659db17a6af71bec57) implementing a basic row with this functionality. You might want to enhance it a bit before using it but feel free to do so.

**Comment by anlaital** [comment]
I've implemented a `ComponentPickerRow` for this purpose in our app using Eureka. Let me know if I can be of help here. :)

**Comment by santiagofm** [comment]
Hey @anlaital! It would be awesome if you can contribute to the [EurekaCommunity](https://github.com/EurekaCommunity) with any row have implemented and consider interesting.
Let me know and i'll add you to the orga!
Cheers

**Comment by cmbasnett** [comment]
@anlaital I would also love to see this!

**Comment by mtnbarreto** [comment]
`TriplePickerInlineRow`, `TriplePickerRow`, `TriplePickerInputRow` provide this functionality.  
https://github.com/xmartlabs/Eureka/pull/1540

**Comment by mtnbarreto** [comment]
done, now tuple properties are public

**Comment by dbmrq** [comment]
@mtnbarreto Awesome, thank you! 🍻
